README.md

```diff
@@ -99,7 +111,9 @@ $("button").on("click", function(event){
 
 Your Answer:
 ```js
-// write code here
+document.getElementsByTagName("button").addEventListener("click", function(event){
+  event.target.style.color = "red";
+})
```

Comment:
✅ 

I'll give it to you, but with vanilla JS you need to loop through all the elements returned by `getElements` and `addEventListener` for each one.